core/src/toga/widgets/splitcontainer.py

```diff
@@ -1,106 +1,118 @@
-import warnings
+from __future__ import annotations
+
+from toga.constants import Direction
 
 from .base import Widget
 
 
 class SplitContainer(Widget):
-    """A SplitContainer displays two widgets vertically or horizontally next to each
-    other with a movable divider.
-
-    Args:
-        id (str):  An identifier for this widget.
-        style (:obj:`Style`): An optional style object.
-            If no style is provided then a new one will be created for the widget.
-        direction: The direction for the container split,
-            either `SplitContainer.HORIZONTAL` or `SplitContainer.VERTICAL`
-        content(``list`` of :class:`~toga.Widget`): The list of components to be
-            split or tuples of components to be split and adjusting parameters
-            in the following order:
-            widget (:class:`~toga.Widget`): The widget that will be added.
-            weight (float): Specifying the weighted splits.
-            flex (Boolean): Should the content expand when the widget is resized. (optional)
-    """
-
-    HORIZONTAL = False
-    VERTICAL = True
+    HORIZONTAL = Direction.HORIZONTAL
+    VERTICAL = Direction.VERTICAL
 
     def __init__(
         self,
         id=None,
         style=None,
-        direction=VERTICAL,
-        content=None,
-        factory=None,  # DEPRECATED!
+        direction: Direction = Direction.VERTICAL,
+        content: list[Widget | tuple[Widget, int]] | None = None,
     ):
+        """Create a new SplitContainer.
+
+        Inherits from :class:`toga.Widget`.
+
+        :param id: The ID for the widget.
+        :param style: A style object. If no style is provided, a default style will be
+            applied to the widget.
+        :param direction: The direction in which the divider will be drawn. Either
+            :attr:`~toga.widgets.SplitContainer.HORIZONTAL` or
+            :attr:`~toga.widgets.SplitContainer.VERTICAL`; defaults to
+            :attr:`~toga.widgets.SplitContainer.VERTICAL`
```

Comment:
None of these symbols are rendered as links, and neither is `Direction` itself. In fact, the documentation doesn't even indicate which package `Direction` is in.

Reply:
I've added documentation for the Constants module, which includes the Direction enum.